docs/en/users/07_Frequently_Asked_Questions.md

```diff
@@ -43,4 +43,24 @@ Since [1.8.0](https://github.com/FreshRSS/FreshRSS/releases/tag/1.8.0) release,
 ```sh
 ./cli/update_user.php --user <username> --password <password>
 ```
-For more information on that matter, there is a [dedicated documentation](../../cli/README.md).
\ No newline at end of file
+For more information on that matter, there is a [dedicated documentation](../../cli/README.md).
+
+## Permissions under SELinux
+
+Some Linux distribution like Fedora or RedHat Enterprise Linux have SELinux system enabled. This acts like a firewall application, so all applications cannot write/modify files under certain conditions. While installing FreshRSS, step 2 can fail if the httpd process cannot write to the following directories : 
+
+ - FreshRSS/data
+ - FreshRSS/data/cache
+ - FreshRSS/data/favicons
+ - FreshRSS/data/users
+
+The following commands should be executed as root :
+
+```sh
+semanage fcontext -a -t httpd_sys_rw_content_t '/usr/share/FreshRSS/data
```

Comment:
I guess it's missing a quote at the end of the line. Same goes in the other file